examples/bri-3/src/bri/merkleTree/capabilities/updateMerkleTree/updateMerkleTree.command.ts

```diff
@@ -0,0 +1,7 @@
+export class UpdateMerkleTreeCommand {
+  constructor(
+    public readonly id: string,
+    public readonly leaves: string[],
+    public readonly hashFunction: string,
```

Comment:
what does hashFunction signify here? Is it going to be a callback or fallback function to be applied on the data?

Reply:
right now its just the name of the hashFunction - i.e. sha256, mimc, etc. Think it should be renamed to hashName? @Manik-Jain

Reply:
yes i think that would be better

Reply:
or hashAlgh

Reply:
Going with hashAlgName  - I dont think hashAlgH will provide clarity seeing as I am not sure what the "h" is for, and people seem confused with the word "function" since it already exists as a high level language key word.